sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.1.2 (2023-09-26)
```

Comment:
Can we change the date here to `unreleased`, the latest released version I see is 1.1.1

Reply:
done